src/react-native/expo-metro.ts

```diff
@@ -28,7 +28,18 @@ export async function addSentryToExpoMetroConfig() {
     return undefined;
   }
 
+  Sentry.setTag('expo-metro-config', 'exists');
+  clack.log.info(`Updating existing ${metroConfigPath}.`);
+
   const mod = await parseMetroConfig();
+  if (!mod) {
+    clack.log.error(
+      `Could read from file ${chalk.cyan(
+        metroConfigPath,
+      )}, please follow the manual steps.`,
+    );
```

Comment:
The `parseMetroConfig` already log the error, we can only show the instructions here.

Reply:
Good catch Krystof 👍 
Removed the duplicate error log with d55407e023747ec31a80facf71f644edae1e4f5a